tests/models/mamba/test_modeling_mamba.py

```diff
@@ -50,16 +50,16 @@ def __init__(
         vocab_size=99,
         hidden_size=32,
         num_hidden_layers=2,
-        intermediate_size=37,
+        intermediate_size=32,
```

Comment:
in MambaConfig, there is self.intermediate_size = int(self.expand * self.hidden_size),not a set attr.

Reply:
Yes, I need to remove the `intermediate_size`